core/src/validator.rs

```diff
@@ -627,6 +694,21 @@ fn new_banks_from_ledger(
         process::exit(1);
     });
 
+    let tower = post_process_restored_tower(
+        restored_tower,
+        &validator_identity,
+        &vote_account,
+        &config,
+        &ledger_path,
+        &bank_forks,
+    );
+
+    info!(
+        "Tower state: root slot={:?}, last vote slot={:?}",
```

Comment:
This is one-off loggging, so I'd rather like to print the whole thing to make debugging easier.